alacritty_terminal/src/config/window.rs

```diff
@@ -117,3 +141,45 @@ impl Dimensions {
         self.columns.0 as u32
     }
 }
+
+/// Window class
+#[derive(Clone, Default, Debug, Eq, PartialEq)]
+pub struct Class {
+    pub instance: Option<String>,
+    pub class: Option<String>
```

Comment:
I think neither instance or class should be optional here. We should probably just fall back to `Alacritty` for both.